src/PufferVault.sol

```diff
@@ -96,7 +96,7 @@ contract PufferVault is
      * @inheritdoc ERC4626Upgradeable
      * @dev Restricted in this context is like `whenNotPaused` modifier from Pausable.sol
      */
-    function mint(uint256 shares, address receiver) public override restricted returns (uint256) {
+    function mint(uint256 shares, address receiver) public virtual override restricted returns (uint256) {
```

Comment:
Where are we planning to override mint function? (usage of virtual override together)

Reply:
This is PufferVault.sol (v1) So I had to add it here so that I can override it in V2 (to add the modifier).